src/main/paradox/docs/api/iam/iam-acls-api.md

```diff
@@ -1 +1,206 @@
-# Access Control List
\ No newline at end of file
+# Access Control Lists
+
+Access Control Lists are rooted in the `/v1/acls` collection.
+
+An ACL defines the applications' data access restriction using the following three parameters:
+         
+- permission: the value used to limit application's access to information.
+- identity: the way to identify individuals.
+- path: the address where to apply the restrictions. Examples of paths are: `/`, `/myorg` or `/myorg/myproject`
+
+Any resources in the system might be protected using an **access token**, provided by the HTTP header `Authorization: Bearer {access_token}`. Visit @ref:[Authentication](./authentication.md) in order to learn more about how to retrieve an access token.
```

Comment:
Update to:
```
Access to resources in the system depends on the access control list set for them. Depending on the access control list, a caller may need to prove its identity by means of an **access token** provided in the `Authorization` header as a bearer token (`Authorization: Bearer {token}`). Please visit @ref:[Authentication](./authentication.md) to learn more about how to retrieve an access token.
```

Reply:
This is way longer. Take into account that this message is shown in every API page (resources, resolvers, projects, views, ...). I don't see why the current one is wrong.

Reply:
The current one is wrong because the resources are not protected _by an access token_, but rather resources are protected (based on the configured ACLs); the client may need to provide an access token for proof of identity.
I don't think it's too long, but if you think it's a big deal, you can probably shorten the second sentence.